src/common/gui/CPatchBrowser.cpp

```diff
@@ -183,10 +207,15 @@ bool CPatchBrowser::populatePatchMenuForCategory( int c, COptionMenu *contextMen
         for (int i = subc * splitcount; i < min((subc + 1) * splitcount, (int)ctge.size()); i++)
         {
             int p = ctge[i];
-            // sprintf(name,"%i. %s",p,storage->patch_list[p].name.c_str());
-            sprintf(name, "%s", storage->patch_list[p].name.c_str());
+
+            #if WINDOWS
+               string patch_name = storage->patch_list[p].name;
+               findReplaceSubstring(patch_name, string("&"), string("&&"));
+            #endif
+
+            name = patch_name;
```

Comment:
Well so obviously this won’t work outside of windows since you only define patch name on windows. Move the string patch name to outside the windows block and Mac and Linux will be ok

Reply:
Yes just noticed that and pushed a fix!

It was a very huge facepalm I made. Then I figured out I don't need `patch_name` at all, I can just use `name` declared beforehand!